public/app/features/alerting/unified/GrafanaRuleQueryViewer.tsx

```diff
@@ -46,6 +47,8 @@ export function GrafanaRuleQueryViewer({
   const expressions = queries.filter((q) => isExpressionQuery(q.model));
   const styles = useStyles2(getExpressionViewerStyles);
 
+  const thresholds = getThresholdsForQueries(queries);
```

Comment:
I noticed an error is thrown inside `getThresholdsForQueries` when you switch from the classic condition to another expression type. No UI visible side effects though

Reply:
Seems to be in `main` too but I was able to track it down and fixed it in https://github.com/grafana/grafana/pull/66797/commits/e43dd950d656d2e18eb335b611ba6c8944c4450b :)